.gitignore

```diff
@@ -5,3 +5,5 @@ coverage.lcov
 .nyc_output
 package-lock.json
 yarn.lock
+es
+lib
```

Comment:
I see you left out `validator.js` and `validator.min.js`; I was actually a little split on whether we should gitignore these. I think I'm okay with this.

Reply:
I think they should be left in. Users can then grab one of them for their front-end assets without having to build it from scratch.